packages/modelviewer.dev/examples/scene-graph.html

```diff
@@ -105,11 +105,162 @@ <h2 class="demo-title">Change Material Base Color</h2>
   </div>
 
 
+  <div class="sample">
+    <div id="demo-container-2" class="demo"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="heading">
+          <h2 class="demo-title">Change Material Metalness and Roughness Factors</h2>
+        </div>
+        <example-snippet stamp-to="demo-container-2" highlight-as="html">
+          <template>
+
+<model-viewer id="sphere" camera-controls interaction-prompt="none" src="../shared-assets/models/reflective-sphere.gltf" alt="A 3D model of a sphere">
+  <div id="controls">
+    <input id="metalness" type="range" min="0" max="1" step="0.01" value="1" />
+    <input id="roughness" type="range" min="0" max="1" step="0.01" value="0" />
+  </div>
+</model-viewer>
+<script>
+const modelViewerParameters = document.querySelector("model-viewer#sphere");
```

Comment:
Let's change the base color of the sphere to something colorful (gold?), as that will make the difference between metal and dielectric more obvious.